gr-analog/examples/fm_demod.py

```diff
@@ -25,7 +25,7 @@ class build_graph(gr.top_block):
     def __init__(self):
         gr.top_block.__init__(self)
 
-        input_rate = 200e3   # rate of a broadcast FM station
+        input_rate = 400e3   # rate of a broadcast FM station
```

Comment:
We're losing a bit resampling fidelity in the polyphase resampler if we don't increase `nfilts` when we increase the resampling ratio, but that's probably irrelevant in practice.

Reply:
I mean, this is correct, the matching TX flow graphs nominally samples at 400 kHz.